src/main/bootstrap/build.gradle

```diff
@@ -0,0 +1,7 @@
+plugins {
+  id "com.github.lkishalmi.gatling" version "3.3.4"
```

Comment:
Note: Will have to change after the gradle portal release ^^

Reply:
This is already mentioned by me in start post of PR.

Reply:
But as you mentioned, the id is already defined.